### ISSUE TITLE
One Light Theme: The highlighted text within comment blocks in the search results has low contrast/not readable

### ISSUE THREAD

**Issue by aybarsnazlica** [zed-industries/zed#36495]
### Summary

One Light theme: The highlighted text within comment blocks in the search results has low contrast and  hard to read. 

### Description

When using the One Light theme,  the highlighted text within comment blocks in the search results has low contrast and  hard to read. 

<img width="823" height="316" alt="Image" src="https://github.com/user-attachments/assets/9dfafd06-3f7e-406f-a74c-960cc1c8a601" />

Similar to #28789.

Steps to reproduce:
1. Use the builtin One Light theme
2. Go to settings and make a search

**Expected Behavior**: The highlighted text within comment blocks in the search results should be readable.
**Actual Behavior**: The highlighted text within comment blocks in the search results is not readable.

### Zed Version and System Specs

Zed: v0.199.10 (Zed) 
OS: macOS 15.6.0
Memory: 32 GiB
Architecture: aarch64

**Comment by michaelkjed** [comment]
It looks okay on my side.

Zed 0.199.10
OS: macOS 15.6

<img width="941" height="699" alt="Image" src="https://github.com/user-attachments/assets/d0391251-691a-4fae-9e1b-0043b2967742" />

**Comment by kotx** [comment]
I am also experiencing this. Happens during search (but not selection like the screenshot above).
